### PR TITLE
Icons and style adjustments

### PR DESCRIPTION
**Pull request by slocker3**
- many adjustments to anchors
-- icons added to linked headers
-- icons added to meta lists
-- inline anchors now have `400` `text-weight`

- lighten the dark gray that has been used as a background `#191919` to `#292929`

- add class `sub-callout` for a subhead feel without using a head

- change icon `svg` `fill` colors

### CODE REVIEW

**Comment by slocker3** [comment]
@jeremyfelt #reviewmerge #bump

File: style-guide/secondary.html [hunk 1]
```diff
@@ -162,7 +162,7 @@
 							<h1>Strategic Communication</h1>
 						</header>
 						<div class="column one" style="">
-							<h4>Department of Strategic Communication</h4>
+							<p class="sub-callout">Department of Strategic Communication</p>
```
Comment (jeremyfelt) [inline style-guide/secondary.html:165]: Is this an editorial instruction in production or do we need to account for this with custom code?

**Review by jeremyfelt — APPROVED** [review]
👍

File: style-guide/secondary.html (hunk 1 above)
Comment (slocker3) [reply style-guide/secondary.html:165]: Right now it is instruction. Are you asking if it should be a class available to added to a paragraph within TinyMCE?

Comment (jeremyfelt) [reply style-guide/secondary.html:165]: Yeah. If that makes sense to help with content entry, it could be something to try.

Comment (slocker3) [reply style-guide/secondary.html:165]: We should ask @allenjohns on Monday if there is any concern with this being something easily available for all content editors to use.

**Pull request merged**